apps/js-machine-front/src/app/components/digestContentLoader/DigestContentLoader.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { memo } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import { TextBlock } from './TextBlock';
+
+const useStyles = makeStyles({
+  root: {
+    width: '100%',
+    animation: '$blink 1s linear infinite',
+  },
+  "@keyframes blink": {
+    '0%': {
+      opacity: 1,
+    },
+    '50%': {
+      opacity: 0.3,
+    },
+    '100%': {
+      opacity: 1,
+    }
+  },
+});
+
+interface Props {
+  isLoading: boolean;
+}
+
+export const DigestContentLoader = memo(({ isLoading }: Props) => {
+  const classes = useStyles();
+  const TEN_TEXT_BLOCKS = 10;
+
+  return (
+    <Box className={classes.root}>
+        { isLoading && Array.from({ length: TEN_TEXT_BLOCKS }, (_, i) => <TextBlock key={i}/>) }
```

Comment:
maybe it's better to generate this array outside of component, it will be generate similar array each time you trigger render. In this case, you don't need const.